.github/workflows/test-pr.yaml

```diff
@@ -27,6 +27,6 @@ jobs:
 
       - name: Smoke test for '${{ matrix.templates }}'
         id: smoke_test
-        uses: ./.github/actions/smoke-test
```

Comment:
This is present.
https://github.com/devcontainers/template-starter/tree/main/.github/actions/smoke-test